test/common/network/listen_socket_impl_test.cc

```diff
@@ -21,6 +22,15 @@ namespace Envoy {
 namespace Network {
 namespace {
 
+TEST(ConnectionSocketImplTest, LowerCaseRequestedServerName) {
+  absl::string_view serverName("www.EXAMPLE.com");
+  std::string expectedServerName("www.example.com");
```

Comment:
nit: absl::string_view expectedServerName or const std::string

Not a big deal in test code :)